rskj-core/src/main/java/co/rsk/net/handler/txvalidator/TxValidatorGasLimitValidator.java

```diff
@@ -32,6 +33,6 @@
     @Override
     public boolean validate(Transaction tx, AccountState state, BigInteger gasLimit, BigInteger minimumGasPrice, long bestBlockNumber) {
         BigInteger txGasLimit = tx.getGasLimitAsInteger();
-        return gasLimit.compareTo(txGasLimit) >= 0;
+        return  txGasLimit.compareTo(gasLimit) <= 0;
```

Comment:
Please use `BIUtil.isMoreThan(gasLimit, txGasLimit)` or `!BIUtil.isLessThan(txGasLimit, gasLimit)`

Reply:
Semantic of those methods is not very clear IMHO

Reply:
Would be great to have extension methods